src/respond-reject.c

```diff
@@ -41,15 +41,24 @@
 #include "util-debug.h"
 #include "util-privs.h"
 
+#include <net/if.h>
+
 int RejectSendIPv4TCP(ThreadVars *, Packet *, void *);
 int RejectSendIPv4ICMP(ThreadVars *, Packet *, void *);
 int RejectSendIPv6TCP(ThreadVars *, Packet *, void *);
 int RejectSendIPv6ICMP(ThreadVars *, Packet *, void *);
 
+static TmEcode RespondRejectInit(ThreadVars *tv, const void *initdata, void **data);
+
+typedef struct RespondRejectThread_ {
```

Comment:
Can this data be different per thread?

Reply:
No, I agree it may be over kill to have a thread dependencies here. My only motivations were:
- not to use a global variable
- be ready for more advanced setup like reject per capture interface